testing/test_aux.out

```diff
@@ -5,14 +5,17 @@ check_style exit code 0
 RESULT pass base.startup.dhcp
 RESULT skip base.switch.ping No local IP has been set, check system config
 RESULT pass base.target.ping target reached
+RESULT pass base.startup.ntp Correct NTP server address
 RESULT skip base.startup.dns No dns traffic detected
 RESULT pass base.startup.dhcp
 RESULT skip base.switch.ping No local IP has been set, check system config
 RESULT pass base.target.ping target reached
+RESULT fail base.startup.ntp Invalid NTP server address
 RESULT fail base.startup.dns Invalid DNS server address
 RESULT pass base.startup.dhcp
 RESULT skip base.switch.ping No local IP has been set, check system config
 RESULT pass base.target.ping target reached
+RESULT fail base.startup.ntp Invalid NTP server address
```

Comment:
I'd either wait on this PR until one of these can be made skip, or file a bug for us to go back and clean it up.

Reply:
fixed it so we handle the skip case.